specifyweb/frontend/js_src/lib/components/WebLinks/spec.ts

```diff
@@ -91,7 +89,10 @@ const argumentSpec = f.store(() =>
       syncers.maybe(syncers.xmlContent),
       syncers.default(localized(''))
     ),
-    // Specify 7 only
+    /**
+     *  Specify 7 only, usage of this feature have been
+     * removed in https://github.com/specify/specify7/issues/4525
+     */
     shouldPrompt: pipe(
```

Comment:
rename this to `legacyShouldPrompt` - that's the naming I used for every syncer prop that is needed fo sp6 but not sps7